src/modules/ordersTable/pure/OrdersTableContainer/index.tsx

```diff
@@ -194,10 +201,12 @@ export function OrdersTableContainer({
             <Trans>
               You don&apos;t have any {isOpenOrdersTab ? 'open' : ''} orders at the moment. <br />
               Create one for free!{' '}
-              <ExternalLink href="https://cow-protocol.medium.com/how-to-user-cow-swaps-surplus-capturing-limit-orders-24324326dc9e">
-                Learn more
-                <ExternalArrow />
-              </ExternalLink>
+              {orderType === TabOrderTypes.LIMIT ? (
```

Comment:
Please, add a TODO to add a link for Advanced orders as well

Reply:
Done